### ISSUE TITLE
Enable the function for a cam that does not support this function

### ISSUE THREAD

**Issue by garry0garry** [JimStar/reolink_cctv#24]
Home Assistant 2022.10.4, Frontend 20221010.0, HACS 1.28.0, Reolink IP NVR/cameras 0.0.12, Reolink RLC-520A

If enable the siren for a cam that does not support the siren, I get an error:

```
Logger: reolink_ip.api
Source: custom_components/reolink_cctv/switch.py:427
Integration: Reolink IP NVR/camera (documentation, issues)
First occurred: 19:40:54 (1 occurrences)
Last logged: 19:40:54

Host 192.168.50.63:80: received an unexpected response while sending command "AudioAlarmPlay": 'value'

```

```
Logger: reolink_ip.api
Source: custom_components/reolink_cctv/host.py:196
Integration: Reolink IP NVR/camera (documentation, issues)
First occurred: 19:40:54 (1 occurrences)
Last logged: 19:40:54

Host 192.168.50.63:80: Unknown exception "" occurred, traceback: Traceback (most recent call last): File "/usr/local/lib/python3.10/site-packages/reolink_ip/api.py", line 2095, in send raise CredentialsInvalidError() reolink_ip.exceptions.CredentialsInvalidError

```

```
Logger: custom_components.reolink_cctv
Source: custom_components/reolink_cctv/host.py:196
Integration: Reolink IP NVR/camera (documentation, issues)
First occurred: 19:40:54 (1 occurrences)
Last logged: 19:40:54

Unexpected error fetching reolink.cam5 data:
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 205, in _async_refresh
    self.data = await self._async_update_data()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 164, in _async_update_data
    return await self.update_method()
  File "/config/custom_components/reolink_cctv/__init__.py", line 87, in async_device_config_update
    await host.update_states() # Login session is implicitly updated here, so no need to explicitly do it in a timer
  File "/config/custom_components/reolink_cctv/host.py", line 196, in update_states
    return await self._api.get_states()
  File "/usr/local/lib/python3.10/site-packages/reolink_ip/api.py", line 765, in get_states
    response = await self.send(body)
  File "/usr/local/lib/python3.10/site-packages/reolink_ip/api.py", line 2095, in send
    raise CredentialsInvalidError()
reolink_ip.exceptions.CredentialsInvalidError

```

**Comment by JimStar** [comment]
Please attach the full debug-log after this happens. You can enable the debug-logging for the component as described in the issue-template...

**Comment by JimStar** [comment]
Just in case: I'm in New Zealand, so there may be some delays with my answers because of very different time of the day...

**Comment by garry0garry** [comment]
Are these the correct logging settings?

```
logger:
  default: warning
  logs:
    custom_components.reolink_cctv: debug
    custom_components.reolink_cctv.switch: warning
    custom_components.reolink_cctv.host: warning
    reolink: debug
```
[home-assistant.copy.log](https://github.com/JimStar/reolink_cctv/files/9793753/home-assistant.copy.log)

**Comment by JimStar** [comment]
No. These are:

```
logger:
  default: warning
  logs:
    homeassistant.core: warning
    custom_components.reolink_cctv: debug
    reolink_ip: debug
```

**Comment by JimStar** [comment]
And don't forget that you need to have the `reolink_dev` **disabled** when using this component... Otherwise they can conflict with each other...  
After that, and after adding these debug-config records in the config - post here please the debug-log, from the very beginning (HA restart).

**Comment by garry0garry** [comment]
reolink_dev is disabled per camera. But I didn't remove reolink_dev from HACS...

Log:
[home-assistant.copy.log](https://github.com/JimStar/reolink_cctv/files/9793805/home-assistant.copy.log)

Now I will remove reolink_dev from HACS. Collect logs again?

**Comment by JimStar** [comment]
This is not a proper debug-log. It doesn't have any debug information, looks like you've not configured the debug-logging properly...

**Comment by garry0garry** [comment]
Then I don't understand how to set it up :-(
![Снимок экрана от 2022-10-16 10-22-56](https://user-images.githubusercontent.com/95581374/196023591-a06cf284-121e-46ac-bccc-23b5674350cf.png)
```
logger:
  default: warning
  logs:
    homeassistant.core: warning
    custom_components.reolink_cctv: debug
    reolink_ip: debug

```

Now I will reboot the computer, and not just HA.

**Comment by JimStar** [comment]
Your screenshot looks correct. But the content of the log-file doesn't have any debug-info.

**Comment by JimStar** [comment]
I remember some people with cameras like yours reported all works without any problems... So it's weird you have so many login-errors with all your cameras all the time (I can see it even in these non-debug logs you've so far posted here)...  
Maybe some problems with your HA installation?.. Or some conflicts of the installed custom-components?.. Try to check this too...

**Comment by garry0garry** [comment]
Removed `reolink_dev` from HACS.
Computer rebooted. Debugging is now visible.
I don't see any major issues with cameras.

Log:
[home-assistant.copy.zip](https://github.com/JimStar/reolink_cctv/files/9793901/home-assistant.copy.zip)

Enabling `switch.camX_email` causes the camera to become unavailable 40% of the time.

**Comment by JimStar** [comment]
Yep, this log has the debug info.  
I'll check this tomorrow, it's already late here...

**Comment by JimStar** [comment]
The issue you've reported here consists of **two** parts. The main one, described by the ticket-title is easy to fix, and this ticket will be closed by this fix soon.  
But the second log-error you've posted here - is another problem, that is the same as in #22. So I will continue there about this second issue...

**Comment by JimStar** [comment]
Should be fixed by **0.0.13**.
